Formula/vcs.rb

```diff
@@ -0,0 +1,29 @@
+class Vcs < Formula
+  desc "Creates video contact sheets (previews) of videos."
+  homepage "https://p.outlyer.net/vcs/"
+  url "https://p.outlyer.net/vcs/files/vcs-1.13.2.tar.gz"
+  sha256 "fc2a2b3994d5ffb5d87fb3dceaa5f6855aca7a89c58533b12fd11b8fb5b623af"
+
+  depends_on "ffmpeg"
+  depends_on "mplayer" => :optional
+  depends_on "gnu-getopt"
```

Comment:
They specifically require "enhanced" `getopt`, which they check for using `getopt -T`.